forks.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+
+const forks = JSON.parse(fs.readFileSync(path.resolve(__dirname, 'forks.json')).toString());
+const tmpDir = path.resolve(__dirname, '.forks-tmp');
+
+if (fs.existsSync(tmpDir)) {
+  execSync(`rm -rf ${tmpDir}`);
+}
+
+fs.mkdirSync(tmpDir);
+
+process.chdir(tmpDir);
+
+forks.map(fork => {
+  const dir = new URL(fork.repository).pathname.substr(1);
+  execSync(`mkdir -p AmazeeLabs`);
+  execSync(`mkdir -p tmp/AmazeeLabs`);
+  execSync(`git clone ${fork.repository} tmp/${dir}`);
```

Comment:
We might need branching support, to target the original branches (ie: `8.x` vs `1.x`...)

Reply:
That should already be handled, since the required branch is defined in `composer.json` in `silverback-drupal`.

Reply:
I think the drawback is that this hides the upstream version that is in use on a client project and forces us to use.
```
    "minimum-stability": "dev",
```
To require the fork. A future improvement might be to raise some kind minor release version every time something changes on the fork so is more obvious when you are updating.

Reply:
I also thought about that. But i don't see a way to create a "sub-version". If we just increase the patch-version, it will collide with the next patch version of the module. If we run our own versioning, we loose all version-range requirements other dependencies have.

But can't we keep global `"minimum-stability": "dev"` and just use `-dev` version constraints when requiring the fork? We can just get every commit there, since every commit of the fork was initiated by a release of the origin.